.github/workflows/ci-mac-on-main-pr.yml

```diff
@@ -101,6 +106,39 @@ jobs:
           CC: clang
           CXX: clang++
 
+      - name: "Running Qiskit tests"
+        run: |
+          virtualenv develop_env
+          source develop_env/bin/activate
+
+          mkdir -p Debug
+          pushd Debug
+          cmake ..
+          make -j qat
+          popd
+
+          git clone https://github.com/microsoft/qiskit-qir.git
+          pushd qiskit-qir
+
+          pip3 install -e .
+          popd
+
+          echo "Testing installation"
+          python -c "from qiskit_qir import to_qir"
+          echo "Done"
+
+          export PYTHONPATH=$PWD/qiskit-qir/tests:$PYTHONPATH
+          export QAT_BINARY=${PWD}/Debug/qir/qat/Apps/qat     
+
+          pushd qir/qiskit-tests/
+          pip3 install pytest
+
+          python3 -m pytest
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CC: clang
+          CXX: clang++
```

Comment:
As I understand the `env:` section sets the environment first, and then the `run:` section executes the commands.
I'm curious, what is the advantage of placing the `env:` section _after_ the `run:`?

Reply:
Your understanding is correct. There is no difference depending on order - the section acts as a dictionary and the sections are used for their respective tasks (i.e. `env` is looked up to set environment, `run` to execute the script)